Automattic/jetpack
# Blocks: remove Amazon block

Fixes #25298

## Proposed changes:

The block was added as a Beta block in 2020, but we never completed it (see #14441).

At this point, it may be better to remove it, since if we ever decide to work on it again we may take a different approach.

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Start with a site connected to WordPress.com.
* Enable Beta blocks with the following snippet:
```php
add_filter( 'jetpack_blocks_variation', function () {
	return 'beta';
} );
```
* Go to Posts > Add New
    * You should not find any Amazon block offered by Jetpack.
